CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+### Bug Fixes
+* Fix handling of Metadata Write message id helper fields.
```

Comment:
Should we have a issue to track these?

Reply:
Typically I would say we should have one yes.  There is some precedent for quick small tweaks leaving that part out so long as the PR is good.